_posts/2025-03-06-Consistency-Check.md

```diff
@@ -0,0 +1,83 @@
+---
+title: JabRef's new GUI for Bibliography Consistency Check
+author: priyanshu
+---
+
+Hi, I’m Priyanshu, an engineering student in Computer Science. In this blog post, I’m glad to introduce you to a new feature that I worked on - a graphical user interface (GUI) for checking bibliography consistency in JabRef.
+
+## Why it matters?
+
+You’re finalizing your research paper for an upcoming conference, and the deadline is near. While reviewing your references, you notice inconsistencies - some citations are missing DOIs, others have incorrect page numbers, and a few lack author names. Manually fixing these across dozens of entries is tedious and time-consuming.
```

Comment:
The feature **does not** fix page numbers. Maybe, just remove this.

Moreover, fixing this is still manual work (although JabRef could be improved to do that manually; we have DOI lookup, fetching information from DOI etc).

Maybe use "Manually finding these incosistency ..." instead of "Manually fixing these..."